### DISCUSSION TITLE
ANSI color tags show up in Annotation output

### DISCUSSION THREAD

**Discussion by StephenHodgson** [community/community#41556] [Actions]
### Select Topic Area

Bug

### Body

In GitHub Actions, annotations are shown on the summary page after the workflow has run. If a log contains ANSI color tags, those tags are not stripped from the annotation. 

![image](https://user-images.githubusercontent.com/13334553/207141039-cbec6c97-cb6b-47de-9598-d71480140c1c.png)

Expected Result:

Annotation text should color the text correctly with the ANSI color tags

**Comment by guilgaly** [comment]
It would be great if something could be done for this - since we can't create an annotation without writing its output to the console, we have to choose between setting nice text colors for warnings/errors/etc. in the console output or creating annotations. 👍
